examples-api-use/flask/static/js/canvas-func.js

```diff
@@ -0,0 +1,99 @@
+
+
+function init_canvas(w, h)
+{
+    
+
+
+    var errMsgDiv = document.getElementById('errMsg');
+    var canvas = document.getElementById('ftCanvas');
+    var ctx = canvas.getContext("2d");
+    var webgl = canvas.getContext("experimental-webgl");
+    function animateFunction(func) {
+        fn = function () {
+            var d = new Date();
+            var t = d.getTime();
+            func(ctx, webgl, t);
+
+            requestAnimationFrame(fn);
+
+        };
+
+        fn();
+
+        return fn;
+    }
+
+    function animFunc(ctx,gl,t,w,h) 
+    {
+        const speed = 0.01
+        const centerX = 20 + Math.sin(t * speed) * 25
+        const centerY = 20
+        const radius = 15
+
+
+        ctx.beginPath();
+        ctx.arc(centerX, centerY, radius, 0, 2 * Math.PI, false);
+        ctx.fillStyle = 'green';
+        ctx.fill();
+        ctx.lineWidth = 1;
+        ctx.strokeStyle = '#003300';
+        ctx.stroke();
+    }
+
+
+    function errFunc(ctx, gl, t, w, h) {

```

Comment:
This is a bit inconsistent: above, you start the brace on a new line, here you put it after the function header. Either is fine, but please stick to one style.